src/platforms/android/migration.mdx

```diff
@@ -6,6 +6,10 @@ redirect_from:
 description: "Migrating between versions of Sentry's SDK for Android."
 ---
 
+## Migrating from `io.sentry:sentry-android 6.14.x` to `io.sentry:sentry-android 6.15.0`
+
+The `sentry-compose-android` and `sentry-compose` integrations now declare their upstream dependencies as `compileOnly`. Make sure to have the `androidx.navigation:navigation-compose`, `androidx.compose.runtime:runtime` and `androidx.compose.ui:ui` dependencies on the classpath if you're using those integrations or the application will crash with a `NoClassDefFoundError` exception.
+
```

Comment:
```suggestion
The `sentry-compose-android` and `sentry-compose` integrations now declare their upstream dependencies as `compileOnly`. Make sure to have the `androidx.navigation:navigation-compose`, `androidx.compose.runtime:runtime`, and `androidx.compose.ui:ui` dependencies on the classpath if you're using those integrations. Otherwise, the application will crash with a `NoClassDefFoundError` exception.

```